lib/patches/active_record/quoting.rb

```diff
@@ -0,0 +1,21 @@
+if ::ActiveRecord::VERSION::STRING >= "5.0.0"
+  module Globalize
+    module ConnectionAdapters
+      module Quoting
+
+        private
+
+        def _quote(value)
+          case value
+            when Array      then
+              "'#{quote_string(YAML.dump(value))}'"
+            else
+              super
+          end
+        end
+      end
+    end
+  end
+
+  ActiveRecord::ConnectionAdapters::AbstractAdapter.include Globalize::ConnectionAdapters::Quoting
```

Comment:
That's right, I removed the DB constants from the environment because it turns out we were just forcing sqlite3 anyway so it was disingenuous.